SpatialGDK/Source/SpatialGDK/Public/Utils/InterestFactory.h

```diff
@@ -35,6 +35,8 @@ class SPATIALGDK_API InterestFactory
 	// Defined Constraint AND Level Constraint
 	Interest CreatePlayerOwnedActorInterest() const;
 
+	void AddActorUserDefinedQueries(const AActor* InActor, const QueryConstraint& LevelConstraints, TArray<SpatialGDK::Query>& OutQueries) const;
+	void RecursivelyAddUserDefinedQueries(const AActor* InActor, const QueryConstraint& LevelConstraints, TArray<SpatialGDK::Query>& OutQueries) const;
```

Comment:
If the goal is to allow us to make as few level constraint changes as possible whenever we need to, why does this function build the set of separate queries rather than build up the constraints to later create the query? Or is there processing of the queries later in order to avoid this complexity?

Reply:
I realise we can't merge queries of different frequencies or result types, however I assume we are expecting duplicates within an actor of some kind across frequency and result type

Reply:
Talked about offline